docs/netlify/_redirects

```diff
@@ -0,0 +1,530 @@
+/docs/:major.:minor/MultipleSelectUI.html /docs/:major.:minor/javascript-data-grid/multipleselectui
+/docs/:major.:minor/NestedHeaders.html /docs/:major.:minor/javascript-data-grid/api/nested-headers/
+/docs/:major.:minor/NestedRows.html /docs/:major.:minor/javascript-data-grid/api/nested-rows/
+/docs/:major.:minor/NodeStructure.html /docs/:major.:minor/javascript-data-grid/api/node-structure/
+/docs/:major.:minor/ObserveChanges.html /docs/:major.:minor/javascript-data-grid/api/observe-changes/
+/docs/:major.:minor/OperatorsComponent.html /docs/:major.:minor/javascript-data-grid/api/operators-component/
+/docs/:major.:minor/Options.html /docs/:major.:minor/javascript-data-grid/api/options/
+/docs/:major.:minor/PersistentState.html /docs/:major.:minor/javascript-data-grid/api/persistent-state/
+/docs/:major.:minor/Queue.html /docs/:major.:minor/javascript-data-grid/api/queue/
+/docs/:major.:minor/RadioInputUI.html /docs/:major.:minor/javascript-data-grid/radioinputui
+/docs/:major.:minor/RecordTranslator.html /docs/:major.:minor/javascript-data-grid/api/record-translator/
+/docs/:major.:minor/RowsMapper.html /docs/:major.:minor/javascript-data-grid/api/rows-mapper/
+/docs/:major.:minor/SamplesGenerator.html /docs/:major.:minor/javascript-data-grid/api/samples-generator/
+/docs/:major.:minor/Search.html /docs/:major.:minor/javascript-data-grid/api/search/
+/docs/:major.:minor/Selection.html /docs/:major.:minor/javascript-data-grid/api/selection/
+/docs/:major.:minor/SelectionCalculations.html /docs/:major.:minor/javascript-data-grid/api/selection-calculations/
+/docs/:major.:minor/SelectionRange.html /docs/:major.:minor/javascript-data-grid/api/selection-range/
+/docs/:major.:minor/SelectUI.html /docs/:major.:minor/javascript-data-grid/selectui
+/docs/:major.:minor/Sheet.html /docs/:major.:minor/javascript-data-grid/api/sheet/
+/docs/:major.:minor/Stack.html /docs/:major.:minor/javascript-data-grid/api/stack/
+/docs/:major.:minor/Storage.html /docs/:major.:minor/javascript-data-grid/api/storage/
+/docs/:major.:minor/TableView.html /docs/:major.:minor/javascript-data-grid/api/table-view/
+/docs/:major.:minor/TextEditor.html /docs/:major.:minor/javascript-data-grid/api/text-editor/
+/docs/:major.:minor/Transformation.html /docs/:major.:minor/javascript-data-grid/api/transformation/
+/docs/:major.:minor/TrimRows.html /docs/:major.:minor/javascript-data-grid/api/trim-rows/
+/docs/:major.:minor/UndoRedo.html /docs/:major.:minor/javascript-data-grid/api/undo-redo/
+/docs/:major.:minor/UndoRedoSnapshot.html /docs/:major.:minor/javascript-data-grid/api/undo-redo-snapshot/
+/docs/:major.:minor/UnmergeCellsAction.html /docs/:major.:minor/javascript-data-grid/api/unmerge-cells-action/
+/docs/:major.:minor/ValueComponent.html /docs/:major.:minor/javascript-data-grid/valuecomponent.html
+ 
+/docs/:major.:minor/tutorial-introduction/ /docs/javascript-data-grid/
```

Comment:
All these redirects should have counterparts for variants without `:major.:minor` parts. For example
`/docs/tutorial-compatibility/ /docs/javascript-data-grid/supported-browsers/`